### ISSUE TITLE
New rule: `no-missing-comment`

### ISSUE THREAD

**Issue by VPenkov** [stylelint/stylelint#1719]
Can we get a rule where there *has* to be a comment at the beginning in each file?

Valid result:
```css
/* This is a comment */
.rule { margin: 0; }
```

Invalid result:
```css
.rule { margin: 0; }
```

**Comment by jeddy3** [comment]
I think this is a [great](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/faq.md#what-is-the-difference-between-a-plugin-and-a-rule) [candidate](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#let-everyone-know) for a plugin as, I assume, [it's tied](https://github.com/stylelint/stylelint/issues/1720) into a methodology of creating styleguides. You'd probably want to call it `stylelint-no-missing-start-of-source-comment` for consistency with our existing `no-missing-end-of-source-newline` rule.

@stylelint/core Any objections to marking this as plugin?

**Comment by evilebottnawi** [comment]
@jeddy3 yep, plugin

**Comment by ntwb** [comment]
Agreed, a plugin I think is the best idea here.

----

Related, but not I'm not sure this is 100% relevent to @VPenkov issues:


There used to be a thing called [CSSDoc (Archive.org)](http://web.archive.org/web/20130701121035/http://cssdoc.net/), the domain is still registered but no content exists, this [article](http://www.devlounge.net/code/getting-started-with-cssdoc-commenting) however still exists and covers much of the CSSDoc syntax for commenting. Anecdotally I see this type of CSS commenting in the wild quite regularly.
